test/arrays/diten/replaceArrayAccess/arrayAlias2.bad

```diff
@@ -1,3 +1,2 @@
 2 6 12 20
 2 6 12 20
-arrayAlias2.chpl:2: error: halt reached - array reference count is negative!
```

Comment:
This modification to the .bad file is fine.  The expected output is still what is in the .good file.